index.js

```diff
@@ -898,7 +898,7 @@ _addPathCloser(path, closer) {
 
 _readdirp(root, opts) {
   if (this.closed) return;
-  const options = Object.assign({type: EV_ALL, alwaysStat: true, lstat: true}, opts);
+  const options = {type: EV_ALL, alwaysStat: true, lstat: true, ...opts};
```

Comment:
Even though we support 8.16+, some folks go against the advice since chokidar runs on their node 8.x machines. Object spread appeared in 8.10 or so, so we shouldn't use it until we switch to node 10.x+ only

Reply:
Oh, OK, I will revert the spread changes then

Reply:
Do you think there are more stuff that came in later 8.x versions? I might make a branch with 8.0.0 to test on Ci temporarily to make sure we don't break anything.

Reply:
https://node.green

Reply:
I tested this on Node.js 8.0.0 and it doesn't work already anyway. https://github.com/XhmikosR/chokidar/runs/263880052#step:5:10

So, perhaps we should change the package.json engines version to 8.10?